messages/rules-command.md

```diff
@@ -1,27 +1,71 @@
 # command.summary
 
-*DRAFT*: Command summary
+List the rules that are available to analyze your code.
 
 # command.description
 
-*DRAFT*: Command description
+You can also view details about the rules, such as the engine it's associated with, tags, and description.
+
+Use this command to determine the exact set of rules to actually analyze your code. The "code-analyzer run" command has the same flags as this command, so once you've determined the flag values for this command that list all the rules you want to run, you specify the same values to the "run" command.
+
+Use the --rule-selector flag to narrow down the list of rules based on specific criteria. For example, you can filter by engine, such as the rules associated with the RetireJS or ESLint engine. Or filter by the severity of the rules, such as high or moderate. Every rule has a tag, such as Recommended, and a name, and you can further filter using both. 
+
+You can combine different criteria using colons to further narrow the list; the colon works as an intersection.  For example, "--rule-selector eslint:Security" lists rules associated only with the Eslint engine that have the Security tag.  To add multiple rule selectors together (a union), specify the --rule-selector flag multiple times, such as "--rule-selector eslint:default --rule-selector retire-js:default:3".
+
+If you specify a set of code files (workspace), this command automatically displays only the rules that are actually relevant. For example, if your workspace contains only JavaScript files, the command doesn't show TypeScript rules. This command doesn't actually inspect the files, it uses just the file extension, such as ".js".
+
+If you've created a custom configuration file that overrides the default rule and engine properties, you can specify it with the --config-file flag. 
 
 # command.examples
 
-*DRAFT*: Command examples
+- List the recommended rules for all engines, which is the default rule selector; use concise table format for the output:
```

Comment:
Not all engines will have rules recommended (on by default). So I'm not how I feel about explicitly saying "for all engines".

Consider also elaborating on this default behavior... maybe:

List the recommended rules using the default concise table format. Specifically selects all rules that have a "Recommended" tag. Any rule overrides that you may have specified in a code-analyzer.yml file from your current folder are applied automatically.

We'll have to get the wording right there... but hopefully you get the idea.